src/main/generic/consensus/transaction/Transaction.js

```diff
@@ -139,7 +139,7 @@ class Transaction {
     toString() {
         return `Transaction{`
             + `senderPubKey=${this._senderPubKey.toBase64()}, `
-            + `recipientAddr=${this._recipientAddr.toBase64()}, `
+            + `recipientAddr=${this._recipientAddr}, `
```

Comment:
I might just have realized that a Transaction's `toString` method is never called in the front end even... Sorry, will correct.